analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Utilities/CopyPasteTokenAnalyzer/Unique.CSharp11.cs

```diff
@@ -0,0 +1,40 @@
+﻿public class Sample
+{
+    public const string Prefix = "Prefix_";
+    public const string Suffix = "_Suffix";
+    public const string Name = $"{Prefix}Name{Suffix}";
+    public const string SingleLineRawString = """"SingleLine"""";
+    public const string MultiLineRawString =
+        """"
+        MultiLine
+        """";
+    public const string InterpolatedSingleLineRawString = $""""SingleLine{Prefix}"""";
+    public const string InterpolatedMultiLineRawString =
+        $$""""
+        MultiLine{{Suffix}}
+        """";
+
+    public void Aaa()
+    {
+        var x = 42;
+        var interpolatedWithWhitespaceToken = $"This literal should be $str but the whitespace between interpolation will not: {x} {x}";
```

Comment:
As far as I can tell this message is misleading, the space is actually a string. See:
![image](https://user-images.githubusercontent.com/56015273/195609803-b8936da7-5180-4b06-a3b2-6e392274dc66.png)